templates/post-install-job.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: post-install-apply-manifest
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded,hook-failed
+spec:
+  template:
+    spec:
+      serviceAccountName: mdai-operator-controller-manager
+      containers:
+        - name: kubectl
+          image: bitnami/kubectl:latest # Use an image with kubectl
```

Comment:
should we use a fixed version instead of latest?

Reply:
I was going to add my changes here, and the version will be updated with the fixed version